Library/Formula/grib-api.rb

```diff
@@ -24,9 +23,16 @@ def install
     ENV.deparallelize
     ENV.no_optimization
```

Comment:
It appears they are not longer needed (or at least, I was able to build the formula on 10.9.5 without those two lines). I'll push an update to remove them.